packages/fxa-settings/src/pages/ConnectAnotherDevice/index.tsx

```diff
@@ -0,0 +1,249 @@
+                      Want to get your tabs, bookmarks, and passwords on another
+                      device?
+                    </p>
+                  </FtlMsg>
+                </div>
+                <div className="flex">
+                  <FtlMsg id="connect-another-device-cad-link">
+                    <Link
+                      to={`/pair?entrypoint=${entrypoint}`}
+                      className="cta-primary cta-xl"
+                    >
+                      Connect another device
+                    </Link>
+                  </FtlMsg>
+                </div>
+                <div className="mt-5">
+                  <FtlMsg id="connect-another-device-not-now-link">
+                    <Link className="link-blue" to="/settings">
+                      Not now
+                    </Link>
+                  </FtlMsg>
+                </div>
+              </>
+            )}
+            {device === Devices.FIREFOX_IOS && (
+              /* user verifies in Fx for iOS, assume they are not signed in */
+              <FtlMsg id="connect-another-device-still-adding-devices-message">
+                <p className="text-sm">
+                  Still adding devices? Sign in to Firefox on another device to
+                  complete set-up
+                </p>
+              </FtlMsg>
+            )}
+            {device === Devices.OTHER_ANDROID && (
+              /*Another android browser, encourage Fx for Android installation */
+              <FtlMsg id="connect-another-device-android-complete-setup-message">
+                <p className="text-sm">
+                  Sign in to Firefox for Android to complete set-up
+                </p>
+              </FtlMsg>
+            )}
+            {device === Devices.OTHER_IOS && (
+              /*Safari or Chrome for iOS, encourage installation of Fx */
+              <FtlMsg id="connect-another-device-ios-complete-setup-message">
+                <p className="text-sm">
+                  Sign in to Firefox for iOS to complete set-up
+                </p>
+              </FtlMsg>
+            )}
+            {device === Devices.OTHER &&
+              /* probably some desktop browser */
+              (isSignIn ? (
+                <FtlMsg id="connect-another-device-still-adding-devices-message">
+                  <p className="text-sm">
+                    Still adding devices? Sign in to Firefox on another device
+                    to complete set-up
+                  </p>
+                </FtlMsg>
+              ) : (
+                <FtlMsg id="connect-another-device-signin-to-complete-message">
+                  <p className="text-sm">
+                    Sign in to Firefox on another device to complete set-up
+                  </p>
+                </FtlMsg>
+              ))}
+            {device === Devices.FIREFOX_DESKTOP && (
+              // TODO: The marketing area will have to be a separate component due to the complexity of the logic.
```

Comment:
I *think* we might not be using the marketing area anymore? Probably something to check with Vesta. Maybe we'll be able to leave this out entirely.

Reply:
🤞 !